src/sourceCommand.ts

```diff
@@ -20,7 +20,7 @@ export type ProgressBar = {
 };
 
 export abstract class SourceCommand extends SfdxCommand {
-  public static readonly DEFAULT_SRC_WAIT_MINUTES = 33;
+  public static readonly DEFAULT_WAIT_MINUTES = 33;
```

Comment:
is the idea that the `mdapi:deploy` command will just have it's own flag that's set to zero (that is, just not using the default)?

Reply:
hmmm good point. for `mdapi:deploy:cancel` in toolbelt it was using the `source:deploy` default, it seemed weird to have this variable mention `SRC` when it's now used in both `source` and `mdapi`

very open to reverting this back

Reply:
I think it's good since 33 seems to be most everywhere.  Just making sure we weren't gonna come back and put a `DEFAULT_MDAPI_WAIT_MINUTES` or whatever.